tools/lint-docstring

```diff
@@ -33,9 +34,47 @@ DOCSTRINGS_FOR_NO_FILE_FOLDERS = {
 BLANK_LINE = f"| {COLUMN_WIDTHS[0]*SPACEBAR}| {COLUMN_WIDTHS[1]*SPACEBAR}| {COLUMN_WIDTHS[2]*SPACEBAR}|\n"
 
 
-def main() -> None:
-    create_file_overview_doc()
-    print(f"Generated {Path(DEVELOPER_DOC).name} successfully.")
+def main(fix_file: bool) -> None:
+    if fix_file:
+        create_file_overview_doc()
+        print(f"Generated {Path(DEVELOPER_DOC).name} successfully.")
+    else:
+        lint_file_overview()
+        print("Successful")
+
+
+def lint_file_overview() -> None:
+    """
+    Used for linting of the developer-file-overview.md document
+    """
+    folder_file_docstring = generate_folder_file_docstrings_dict()
+    computed_doc_dict = extract_docstrings_from_file_overview()
```

Comment:
What exactly is "computed" here? Does `existing_doc_dict` sound better. I'm not sure though. What do others think here?